src/routes/open/components/SafeOwnersConfirmationsForm/index.tsx

```diff
@@ -224,6 +224,9 @@ const SafeOwnersForm = (props): React.ReactElement => {
                     }
                   }
                   name={addressName}
+                  type="text"
+                  autoCorrect={false}
+                  spellCheck={false}
```

Comment:
You have already set this attribute on the generic AddressInput component, you do not need to repeat it here. If you want to add a different attribute (autoCorrect), please add it to the generic component

Reply:
Fixed it!